### PR TITLE
Portfolio pop up

### PR DESCRIPTION
**Pull request by dialite**
In this Project I did the following;

-  Use basic JavaScript syntax.
-  Use JavaScript to manipulate DOM elements.
-  Use JavaScript events.
-  Understand how to parse a Figma design to create a UI.
-  Use objects to store and access data.

### CODE REVIEW

**Comment by dialite** [comment]
Expecting your feedback. This is the most demanding project I have ever done. Happy review. Thanks.

File: script.js [hunk 1]
```diff
@@ -1,8 +1,260 @@
...
+  cards.appendChild(card);
+}
+
+createCard(projects, 0);
+createCard(projects, 1);
+createCard(projects, 2);
+createCard(projects, 3);
+createCard(projects, 4);
+createCard(projects, 5);
+createCard(projects, 6);
```
Comment (tugsem) [inline script.js]: - [ ] For best javascript practices, please remove these lines of code and istead, write a loop for calling createCard() function for each element in your projects array.

File: style.css [hunk 2]
```diff
@@ -603,6 +603,196 @@ h2 {
...
+  background-color: #fff;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  box-sizing: border-box;
+  padding: 10px;
+  max-height: 554px;
+  overflow-y: scroll;
+}
```
Comment (tugsem) [inline style.css]: - [ ] Kindly give more height as in the design to your pop-up window. You should remove line 621 as it gives a short height and make it scrollable. Right now, your popup seems like this: 
image
image
on mobile screens: 
image

figma design: 
image
image

**Review by tugsem — CHANGES_REQUESTED** [review]
Hi @dialite >, 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!
Alt Text
## Highlights ⭐ 
✔️ linters 
✔️ Good pr description
✔️ Almost done

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dialite** [comment]
Thanks for the feedback. I have made all corrections

**Review by victorokoroji — APPROVED** [review]
Hi @dialite, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**